tests/integration/test_rex.py

```diff
@@ -639,3 +639,30 @@ def test_date_functions(c):
             FROM df
             """
         )
+
+
+def test_totimestamp(c):
+    df = pd.DataFrame(
+        {
+            "a": [1203073300, 1406073600, 2806073600],
+        }
+    )
+    c.create_table("df", df)
+
+    df = c.sql(
+        """
+        SELECT to_timestamp(a) AS date FROM df
```

Comment:
Maybe I'm missing something, but based on the `OPERATION_MAPPING` shouldn't this be

```suggestion
        SELECT totimestamp(a) AS date FROM df
```